__tests__/features/precision_spec.js

```diff
@@ -291,7 +299,7 @@ describe('#precision', function () {
             Helper.trigger(cancel, 'mouseover');
 
             // then
-            expect(this.target).toHaveHtml(raty.opt.cancelHint);
+            expect(testContext.target[0]).toContainHTML(raty.opt.cancelHint);
```

Comment:
Could we use the simplest selector possible like `innerHTML`? So we depend less on the framework.